src/main/java/seedu/duke/exceptions/NoSuchModuleException.java

```diff
@@ -1,7 +1,9 @@
 package seedu.duke.exceptions;
 
+import seedu.duke.ui.TextUi;
+
 public class NoSuchModuleException extends ModHappyException {
-    private static final String ERROR_MESSAGE = "Sorry, no such module exists ._.";
+    private static final String ERROR_MESSAGE = TextUi.ERROR_NO_SUCH_MODULE;
```

Comment:
At first I thought this was unnecessary because we can just use TextUi.ERROR_NO_SUCH_MODULE in the super() constructor, but after a while I realised that it's a bit more readable this way. It's a good idea.